lib/bin/lwm2m_carrier/os/lwm2m_carrier.c

```diff
@@ -30,8 +30,6 @@ void lwm2m_carrier_thread_run(void)
 
 #ifdef CONFIG_LWM2M_CARRIER_USE_CUSTOM_BOOTSTRAP_URI
 	config.bootstrap_uri = CONFIG_LWM2M_CARRIER_CUSTOM_BOOTSTRAP_URI;
-#endif
-#ifdef CONFIG_LWM2M_CARRIER_USE_CUSTOM_BOOTSTRAP_PSK
 	config.psk = (char *)bootstrap_psk;
```

Comment:
The format has changed so you should create a new Kconfig for the bootstrap PSK.